README.md

```diff
@@ -77,7 +77,7 @@ You can then load the module in GHCi, and run it:
 ```
 
  [hackage]: http://hackage.haskell.org/package/hedgehog
- [hackage-shield]: https://img.shields.io/badge/hackage-v0.5-blue.svg
+ [hackage-shield]: https://img.shields.io/badge/hackage-v0.6.1-blue.svg
```

Comment:
Is there a way to make this dynamic so we don't forget to update it every time there's a version bump?

Reply:
Unfortunately their parser is broken for cabal files and requires the version to be written on the same line as `version:`. See #72